DirectEmployers/MyJobs
# [PD 2981] clarify ui partners contacts

## What's this PR do?
This PR Clarifies the fact that changes made to adding a partner and contact will not be saved until the form is done and submitted

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of NUO and go to add a Partner or Contact

## How should this be manually tested?
Pull Down the repository and go to NUO. You should add a partner and then just follow your way through the form and you will see it will continue to add the items in the side menu that says Pending Details

## Any background context you want to provide?
Jason Sole has approved the design and functionality
